ARTICLE-3.markdown

```diff
@@ -123,8 +128,9 @@ those Org Areas in specific domains.
 Org Areas shall have Lead positions who are responsible for the day-to-day
 operations of the Org Area responsibilities as well as forming the Working
 Groups (see below) necessary for decisions beyond the scope of day-to-day
-operations. Both the Org Area responsibilities, authorities, and Lead shall
-be agreed upon by consensus of the membership.
+operations. The Org Area's responsibilities and authority shall be agreed
+upon by consensus of the membership. The Org Area's Lead position shall be
+a Member-Owner, and shall be agreed upon by consensus of the membership.
```

Comment:
@brodavi , don't we want to say that org area lead can be a non-member owner as well?